### DISCUSSION TITLE
Welcome to picamera2 Discussions!

### DISCUSSION THREAD

**Discussion by davidplowman** [raspberrypi/picamera2#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by klemva** [comment]
Hi team, is any term when we can look forward to a stable release for picamera2? Thank you for answer:)

Václav

**Comment by davidplowman** [comment]
Hi, I'm hoping to post an update on the forum very soon now. Just want to get a couple of final things sorted out and ensure that everything installs properly from apt/pip (functionally it will be the same as what is here now). At this point the release will be "more stable". I don't want promise "completely stable" until we've had some more users and feedback, but the bar on changes that break existing APIs will definitely go up a notch.

**Reply by supertobi** [reply]
I know estimations are always hard, but what is "very soon"? Days, weeks or months?

**Reply by davidplowman** [reply]
Actually we've just pushed everything up to git/PyPI and I've just gone through the "install on a clean Bullseye" which appears to have worked fine, so it's looking good to me at the moment. Hopefully I'll post a short forum update this afternoon.

**Comment by WafaaEll** [comment]
how can i solve this : ModuleNotFoundError: No module named 'libcamera'      when i use this code with raspberry pi 4 model B and camera rev1.3 i installed everything needed and still same prblm 
code :
from picamera2 import PiCamera2
from time import sleep

camera = PiCamera()

camera.start_preview()
sleep(5)
camera.stop_preview()

**Reply by davidplowman** [reply]
> how can i solve this : ModuleNotFoundError: No module named 'libcamera' when i use this code with raspberry pi 4 model B and camera rev1.3 i installed everything needed and still same prblm code : from picamera2 import PiCamera2 from time import sleep
> 
> camera = PiCamera()
> 
> camera.start_preview() sleep(5) camera.stop_preview()

You need
```
from picamera2 import Picamera2

camera = Picamera2()
```
Please refer to the [manual](https://datasheets.raspberrypi.com/camera/picamera2-manual.pdf) and [examples](https://github.com/raspberrypi/picamera2/tree/main/examples) for more information.
